<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QWidget" name="Interface">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="title">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Duck Game Map Editor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="editorModeDropdown">
         <property name="placeholderText">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="preview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGraphicsView" name="previewView"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="selectorDropdown">
         <property name="placeholderText">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
        <property name="orientation">
        <enum>Qt::Vertical</enum>
        </property>
        <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
        </sizepolicy>
        </property>
        <property name="sizeHint">
        <size>
            <width>20</width>
            <height>40</height>
        </size>
        </property>
        </spacer>
        </item>
       <item>
        <widget class="QPushButton" name="exportButton">
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
      </layout>
 </widget>
</ui>
